lib/pulse_message.js

```diff
@@ -3,16 +3,38 @@
 const _compact = require('lodash/compact')
 const _flatten = require('lodash/flatten')
 const _isArray = require('lodash/isArray')
+const _isString = require('lodash/isString')
+const _isEmpty = require('lodash/isEmpty')
 const numberValidator = require('./validators/number')
 const dateValidator = require('./validators/date')
 const stringValidator = require('./validators/string')
 const PublicPulseProfile = require('./public_pulse_profile')
 const Model = require('./model')
 
+const pulseMessageValidator = {
+  id: stringValidator,
+  mts: dateValidator,
+  userID: stringValidator,
+  title: stringValidator,
+  content: stringValidator,
+  isPin: numberValidator,
+  isPublic: numberValidator,
+  tags: stringValidator,
+  attachments: stringValidator,
+  likes: numberValidator,
+  userLiked: numberValidator,
+  comments: numberValidator
+}
+
+const pulseCommentValidator = {
+  ...pulseMessageValidator,
+  parent: stringValidator
+}
+
 const fields = {
   id: 0,
   mts: 1,
-  // placeholder
+  parent: 2,
```

Comment:
i am still not able to receive a message from the api where the 3rd item denotes the parent id. are you sure https://api-pub.bitfinex.com/v2/pulse/hist returns them?